franklin/builder/models.py

```diff
@@ -8,16 +8,16 @@ class Site(models.Model):
     """ Represents a 'deployed' or soon-to-be deployed static site.
 
     :param id: A unique site identified
+    :param repo_name: The full repo name of project on github
     :param git_hash: The git hash of the deployed code
-    :param repo_name: The repo name of the site to check for 're-deploys'
     :param path: The path of the site on the static server
     """
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    git_hash = models.CharField(max_length=50)
-    repo_name = models.CharField(max_length=20, default='')
+    repo_name = models.CharField(max_length=100)
```

Comment:
Ok that sounds good. Let's kill those unneeded fields sooner rather than later though.